src/components/Header.tsx

```diff
@@ -1,20 +1,39 @@
+"use client";
+
 import Image from "next/image";
 import Link from "next/link";
 
+import { useUser } from "@/components/contexts/UserContext";
+
 const headerContainerStyles =
   "flex flex-row items-center justify-between text-awesomer-purple h-36 bg-white px-8";
 
-const isSignedIn = true;
-const hasTeam = true;
-
 export default function Header() {
+  const user = useUser().currentUser;
+
   return (
     <div className={headerContainerStyles}>
       <div className="flex w-48 font-semibold">
-        {isSignedIn ? (
-          hasTeam && <Link href="/">Join a Team</Link>
+        {user.type === "Participant" ? (
```

Comment:
For these comparisons, can we compare to the UserType enum? This has better type safety than comparing it to a string when we already have it defined. (UserType exists in the userContext)
```suggestion
        {user.type === UserType.Participant ? (
```